tutorials/transformations_02_coordinate_system_manager.ipynb

```diff
@@ -9,18 +9,6 @@
     "## Imports"
```

Comment:
<p>a "member function" is a method, right?</p><p>I'd use a enumeration for the parameters, to make the section more readable.</p><p> </p><p>Please explain the concept of a reference system before using it for the parameter explanation. E.g. show an equation to make things clear or just write one or two sentences what's the actual relation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>done</p>